deps/wazuh_testing/wazuh_testing/qa_docs/lib/index_data.py

```diff
@@ -54,7 +54,7 @@ def read_files_content(self, files):
         brief: Opens every file found in the path and appends the content into a list.
         """
         for file in files:
-            with open(os.path.join(self.path, file)) as f:
+            with open(file) as f:
```

Comment:
Try not to use single-letter variables. The name of the variable must always be representative of its content